pages/index.tsx

```diff
@@ -46,7 +48,13 @@ const Home: NextPage = () => {
         console.log("Edge function returned.");
 
         if (!response.ok) {
-            throw new Error(response.statusText);
+          setError(true);
```

Comment:
@mael-queau this is good, however - please create errorHandler function to take care of the setters, it is a better practice. Thank you!